src/SFA.DAS.Commitments.Api/Orchestrators/ApprenticeshipsOrchestrator.cs

```diff
@@ -0,0 +1,285 @@
+﻿using System;
```

Comment:
Can you revert the rename of file and rename the class instead please.
We loose the history otherwise. 
So **ApprenticeshipOrchestrator**
/c

Reply:
Yeah, I was under the impression the history would not be lost, but as it is then happy to do this.

Reply:
Sure. If the file content is unchanged then the history should remain otherwise it seem to think its a new file. 
I would guess that singular is the way to go but we do mix and match a bit..  hehe